src/options.rs

```diff
@@ -92,6 +93,7 @@ pub struct ConfigFlags {
     pub network_use_log: Option<bool>,
     pub network_use_binary_prefix: Option<bool>,
     pub enable_gpu_memory: Option<bool>,
+    pub title_has_hostname: Option<bool>,
```

Comment:
nit: Sorry for suggesting something else, but maybe `title_shows_hostname`? Change the config file and clap option as well.